pkg/apps/blackduck/components/utils/pvc.go

```diff
@@ -0,0 +1,49 @@
+package utils
+
+import (
+	horizonapi "github.com/blackducksoftware/horizon/pkg/api"
+	"github.com/blackducksoftware/horizon/pkg/components"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func CreatePVC(name string, requestedSize string, defaultSize string, storageclass string, accessMode horizonapi.PVCAccessModeType, label map[string]string, namespace string, volumeName string) (*components.PersistentVolumeClaim, error) {
+	// Workaround so that storageClass does not get set to "", which prevent Kube from using the default storageClass
+	var class *string
+	if len(storageclass) > 0 {
+		class = &storageclass
+	} else {
+		class = nil
+	}
+
+	var size string
+	_, err := resource.ParseQuantity(requestedSize)
+	if err != nil {
+		size = defaultSize
```

Comment:
if requestedSize is wrong there's no guarantee defaultSize is any better.  IMO, get rid of defaultSize and just take a size.  If it is invalid, return an error